tools/isobuild/bundler.js

```diff
@@ -3220,7 +3220,51 @@ Find out more about Meteor at meteor.com.
  */
 
 exports.bundle = Profile("bundler.bundle", function (options) {
-  return files.withCache(() => bundle(options));
+  return files.withCache(async () => {
+
+    const buildCache = new Map();
+    
+    const clientOptions = {
+      ...options,
+      buildOptions: {
+        ...options.buildOptions,
+        webArchs: options.buildOptions?.webArchs || 
+                  options.projectContext.platformList.getWebArchs(),
+        serverArch: undefined,
+      },
+      isClientOnlyBundle: true,
+      buildCache,
+    };
+    
+    const serverOptions = {
+      ...options,
+      buildOptions: {
+        ...options.buildOptions,
+        webArchs: [],
+        serverArch: options.buildOptions?.serverArch || archinfo.host(),
+      },
+      isServerOnlyBundle: true,
+      buildCache,
+    };
+    
+    // TODO: we must to paralelize it in two different threads
+    const clientResult = await bundle(clientOptions);
+    const serverResult = await bundle(serverOptions);
+    
+    const combinedResult = {
+      errors: clientResult.errors || serverResult.errors,
+      warnings: clientResult.warnings || serverResult.warnings,
```

Comment:
Should not be like `postStartupCallbacks`? so that if both client and server has errors or warnings both get reported?